Resolve #2874 (#2957)
MicrosoftDocs/windows-powershell-docs: docset/winserver2012-ps/international/Get-WinUserLanguageList.md

## OUTPUTS

### System.Collections.Generic.List<Microsoft.InternationalSettings.Commands.WinUserLanguage>
This cmdlet returns a list of **WinUserLanguage** objects that contain one or more languages and associated properties from the current user account's language list.

> For information about the **Generic.List** object, see [List<T> Class](https://go.microsoft.com/fwlink/?LinkID=243342).

Reviewer: ```suggestion
For information about the **Generic.List** object, see [List(Of T) Class](https://go.microsoft.com/fwlink/?LinkID=243342).
```

> For information about the **Generic.List** object, see [List(Of T) Class](https://go.microsoft.com/fwlink/?LinkID=243342).

Reviewer: ```suggestion
For information about the **Generic.List** object, see [List<T> Class](/dotnet/api/system.collections.generic.list-1).
```

> For information about the **Generic.List** object, see [List(Of T) Class](https://go.microsoft.com/fwlink/?LinkID=243342).

Reviewer: ```suggestion
For information about the **Generic.List** object, see [List<T> Class](https://go.microsoft.com/fwlink/?LinkID=243342).
```
